tests/tailer/savedown_withcount/tests.csv

```diff
@@ -0,0 +1,16 @@
+action,ms,bytes,lang,code,repeate,minver,comment
+before,0,0,q,tr:.servers.gethandlebytype[`tr_seg1;`any],1,,"Connect to tailreader handle to give access the tailreader1 for testing tailer1"
+before,0,0,q,tl:.servers.gethandlebytype[`tailer_seg1;`any],1,,"Connect to tailer1 handle,so tailer can be tested through tl handle"
+before,0,0,q,tl".wdb.numrows:100",1,,"Set row count as the standard row count is 10000 would be to large for a test"
+run,0,0,q,(tr"count tables[]")=2,1,,"Check the number of tables in the tailreader is currently 2"
```

Comment:
this seems very config dependent etc., I'm not sure we should be failing a test if e.g. there are more than 2 tables?